plugins/modules/content_view_filter_rule_info.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021 Paul Armstrong
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: content_view_filter_rule_info
+version_added: 3.9.0
+short_description: Fetch information about a Content View Filter Rule
+description:
+  - Fetch information about a Content View Filter Rule
+author:
+  - "Paul Armstrong @parmstro"
+extends_documentation_fragment:
+  - theforeman.foreman.foreman
+  - theforeman.foreman.foreman.infomodule
+'''
+
+EXAMPLES = '''
+- name: "Show a content_view_filter_rule"
+  theforeman.foreman.content_view_filter_info:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    content_view: "SOE_RHEL9"
+    content_view_filter: "NoFireFox"
+    name: firefox
+
+'''
+
+RETURN = '''
+content_view_filter_rule:
+  description: Details about the found content_view_filter_rule
+  returned: success and I(name) was passed
+  type: dict
+'''
+
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import (
+    ForemanInfoAnsibleModule,
+)
+
+
+class KatelloContentViewFilterRuleInfo(ForemanInfoAnsibleModule):
+    pass
```

Comment:
Please use the `KatelloInfoAnsibleModule` base class, that will make the `organization` parameter mandatory.

And then you'd use the following in the `DOCUMENTATION` block above:
```yaml
extends_documentation_fragment:
  - theforeman.foreman.foreman
  - theforeman.foreman.foreman.katelloinfomodule
  - theforeman.foreman.foreman.infomodule
```
The `katelloinfomodule` needs to come before the normal `infomodule`